backend/quill/src/main/scala/io.buildo.toctoc/quill/authentication/token/QuillAccessTokenAuthenticationDomain.scala

```diff
@@ -34,7 +35,7 @@ class QuillAccessTokenAuthenticationDomain[D <: SqlIdiom, N <: NamingStrategy, C
   private implicit val instantEncoder: Encoder[Instant] =
     encoder[Instant](SqlTypes.TIMESTAMP_WITH_TIMEZONE)
   private implicit val instantDecoder: Decoder[Instant] =
-    decoder[Instant]({ case i: Instant => i }, SqlTypes.TIMESTAMP_WITH_TIMEZONE)
+    decoder[Instant]({ case d: JodaDateTime => d.toDate.toInstant }, SqlTypes.TIMESTAMP_WITH_TIMEZONE)
```

Comment:
I don't understand what is happening here, why Joda?

Reply:
Joda is used internally by quill, so we're given a `JodaDateTime`